h2o-dist/index.html

```diff
@@ -636,6 +665,91 @@ <h2>Gradle-style specification for Maven artifacts</h2>
 &nbsp;compile "ai.h2o:h2o-web:${h2oProjectVersion}"</br>
 }</br>
           </div>
+            <div id="quickstart-kubernetes" style="display:none">
+                <h2>Get started with H<sub>2</sub>O on Kubernetes</h2>
+                <p>1. Set-up kubernetes cluster and kubectl.</p>
+                <p>2. (Optional) Adjust the 'default' namespace in the following YAML, if required.</p>
+    
+                <button class="btn copy_button" id="btnCopy7" data-copied-hint="Copied!" data-clipboard-target='to_copy7'>
+                    <span class="octicon octicon-clippy"></span>
+                </button>
+                <p class="terminal" id="to_copy7">
+apiVersion: apps/v1<br/>
+kind: StatefulSet<br/>
+metadata:<br/>
+  name: h2o-cluster-stateful-set<br/>
+  namespace: default<br/>
+spec:<br/>
+  serviceName: h2o-service<br/>
+  podManagementPolicy: "Parallel"<br/>
+  replicas: 1<br/>
+  selector:<br/>
+    matchLabels:<br/>
+      app: h2o-cluster<br/>
+  template:<br/>
+    metadata:<br/>
+      labels:<br/>
+        app: h2o-cluster<br/>
+    spec:<br/>
+      containers:<br/>
+        - name: h2o-cluster<br/>
+          image: 'h2oai/h2o-open-source-k8s:latest'<br/>
```

Comment:
I think this is technically not correct because the download page should only refer to a specific version. Eg. when you go on the download page for 3.30.0.1 you should only get version 3.30.0.1.

How about nightly builds? Will this be available for nightlies as well?

Reply:
Sure. the version is available in the `SUBST_PROJECT_VERSION` placeholder- I'll fix it. If the format is the `nightly` format, then it'll just substitute it with `nightly` - sounds like a plan ?

Reply:
Sounds good, thank you!

Reply:
Done. Please review. This is hard to automatically test ATM, therefore I did a manual test with manually filled version. 

I utilize the fact that nightlies have at least 4 digits at the end (correct me if I'm wrong). If it's a nightly, then the docker tag is changed to  `nightly` from the original version.

![Screenshot from 2020-08-13 10-27-49](https://user-images.githubusercontent.com/8769110/90112242-050bc100-dd50-11ea-8b76-16b3b837662e.png)
![Screenshot from 2020-08-13 10-28-05](https://user-images.githubusercontent.com/8769110/90112245-05a45780-dd50-11ea-9982-a46d5e37d5a6.png)